ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -69,6 +69,20 @@ tests:
   container:
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm
```

Comment:
`tests migration from in-tree cloud provider to CCM on a running cluster`
I'm not sure if this job is going to be helpful, as Windows Nodes are added only after the cluster is fully up and running, as part of the `windows-e2e-operator-test` pod.